README.md

```diff
@@ -476,18 +481,50 @@ Anatomy of a Redis command
 
 All the Redis commands are defined in the following way:
 
-    void foobarCommand(client *c) {
-        printf("%s",c->argv[1]->ptr); /* Do something with the argument. */
-        addReply(c,shared.ok); /* Reply something to the client. */
+```c
+void foobarCommand(client *c) {
+    printf("%s",c->argv[1]->ptr); /* Do something with the argument. */
+    addReply(c,shared.ok); /* Reply something to the client. */
+}
+```
+
+There will be a `foobar.json` JSON file to record all the metadata for this command:
+
+```json
+{
+    "FOOBAR": {
+        "summary": "A foobar command.",
+        "complexity": "O(1)",
+        "group": "connection",
+        "since": "x.0.0",
+        "arity": 2,
+        "function": "foobarCommand",
+        "command_flags": [
+            "LOADING",
+            "FAST"
+        ],
+        "acl_categories": [
+            "CONNECTION"
+        ],
+        "arguments": [
+            {
+                "name": "message",
+                "type": "string"
+            }
+        ]
```

Comment:
i wonder if maybe this is too verbose and we need to just mention each command has a json file and not give an example?
people can easily just go look at the pingCommand and it's ping.json.
maybe it's better to just replace this whole section with an advise to go look at these two?

Reply:
yes, it's too verbose, i removed it and replace it with the ping advise